### ISSUE TITLE
Cobalt strike

### ISSUE THREAD

**Issue by kushal-17** [ssooking/AutoBeef#2]
Sorry will u plzzz give alternate link for Cobalt strike 3.12 cracked plzz in GitHub it's dmca take down plzzz sir ...

**Comment by iNoSec** [comment]
yeah same here :D i find the pan.baidu.com but cant download because cant complete chinese captcha  :(

**Comment by kushal-17** [comment]
Sir,
      Nope I don't want or tell you to help when I want. but right now I
don't want anything.if you want I will upload. It Cobalt strike 3.12 Mac os
version,windows, Linux,original file.

**Comment by kushal-17** [comment]
Hey,
        Can u heple with metasploit pro crack, innuendo framework,
canvas.please if any of one.
